client/verta/verta/deployment/update/rules.py

```diff
@@ -54,6 +54,24 @@ class MaximumAverageLatencyThresholdRule(_UpdateRule):
     """
     Rule for maximum average latency threshold.
 
+    The JSON equivalent for this is:
+
+    .. code-block:: json
+
+        {
+            "rules": [
+                {
+                    "rule": "latency_avg_max",
+                    "rule_parameters": [
+                        {
+                            "name": "threshold",
+                            "value": "0.1"
+                        }
+                    ]
+                }
+            ]
+        }
```

Comment:
Similar to https://github.com/VertaAI/modeldb/pull/1329#discussion_r474103106, I think we could reduce update rules to fewer lines like we have for autoscaling metrics

Reply:
Done.